simulation/test_dht_storage.cpp

```diff
@@ -75,6 +75,19 @@ namespace
 		TORRENT_ASSERT(!hash.fail());
 		return ret;
 	}
+
+	std::unique_ptr<dht_storage_interface> create_default_dht_storage(
+		dht_settings const& sett)
+	{
+		std::unique_ptr<dht_default_storage> s(dht_default_storage_constructor(sett));
+		TEST_CHECK(s.get() != NULL);
+
+		std::vector<node_id> node_ids;
+		node_ids.push_back(to_hash("0000000000000000000000000000000000000200"));
+		s->update_node_ids(node_ids);
```

Comment:
these days you can actually say:

s->update_node_ids({to_hash("0000000000000000000000000000000000000200")});